test/e2e/cluster/cfn-templates/setup-cfn.yaml

```diff
@@ -362,6 +362,14 @@ Resources:
                 path: /root/download-private-key.sh
                 permissions: "0755"
 
+  PodIdentityS3Bucket:
+    Type: AWS::S3::Bucket
+    DeletionPolicy: Delete
+    Properties:
+      Tags:
+        - Key: !Ref TestClusterTagKey
+          Value: !Ref ClusterName
```

Comment:
could we create the object in s3 from here as well so we don't have to handle it separetly? (`S3Object`)